identifiers/country-ec/provinces.csv

```diff
@@ -23,4 +23,7 @@ ocd-division/country:ec/province:se,Santa Elena
 ocd-division/country:ec/province:sd,Santo Domingo de los Tsáchilas
 ocd-division/country:ec/province:u,Sucumbíos
 ocd-division/country:ec/province:t,Tungurahua
-ocd-division/country:ec/province:z,Zamora Chinchipe
\ No newline at end of file
+ocd-division/country:ec/province:z,Zamora Chinchipe
+ocd-division/country:ec/special:america_latina_el_caribe_y_africa,"America Latina, El Caribe y Africa"

```

Comment:
These special OCD IDs exist primarily for electoral districts in the National Assembly right? I think they should go into the districts.csv rather than provinces.csv

I'm also wondering if `special_ed` would be more descriptive than `special` for these.

Reply:
For other countries, we used **special** for voters living aboard. I agree that in this case, we have multiple districts for people living aboard. 

Note sure if there is a conventional way to do it, so I'll be curious to know what is the recommendation in that case.

Reply:
@jpmckinney @jloutsenhizer  any thoughts ?

Reply:
I think `special` is fine, since we use it for `special:diaspora` in the case where there is a single division for the entire diaspora.

In France, we have `special:diaspora/ed:1`, etc. We could do the same here, e.g. `special:diaspora/ed: america_latina_el_caribe_y_africa`.

I guess a different model could have been `diaspora:general` for single divisions and `diaspora:america_latina_el_caribe_y_africa` for multiple divisions, but that's not what we did.

We don't consistently suffix `ed` for electoral districts; I don't see a strong reason to start.

Reply:
I like the special:diaspora/ed: america_latina_el_caribe_y_africa. 

I just updated the identifiers, please have a look.